app/experimenter/experiments/api/v5/query.py

```diff
@@ -1,31 +1,39 @@
 import graphene
 
-from experimenter.experiments.api.v5.types import NimbusExperimentType
+from experimenter.experiments.api.v5.types import (
+    NimbusExperimentIdOrder,
+    NimbusExperimentOptionalStatus,
+    NimbusExperimentType,
+)
 from experimenter.experiments.models.nimbus import NimbusExperiment
 
 
 class Query(graphene.ObjectType):
-    experiments_by_status = graphene.Field(
-        graphene.List(NimbusExperimentType),
-        status=graphene.Argument(
-            NimbusExperimentType._meta.fields["status"]._type, required=False
-        ),
-    )
-    all_experiments = graphene.List(
+    experiments = graphene.List(
         NimbusExperimentType,
+        description="List Nimbus Experiments.",
+        offset=graphene.Int(),
+        limit=graphene.Int(),
+        order=NimbusExperimentIdOrder(),
+        status=NimbusExperimentOptionalStatus(),
     )
     experiment_by_slug = graphene.Field(
-        NimbusExperimentType, slug=graphene.String(required=True)
+        NimbusExperimentType,
+        description="Retrieve a Nimbus experiment by its slug.",
+        slug=graphene.String(required=True),
     )
 
-    def resolve_all_experiments(root, info):
-        return NimbusExperiment.objects.all()
+    def resolve_experiments(
+        root, info, offset=0, limit=20, order="ascending", status=None
+    ):
+        q = NimbusExperiment.objects
+        q = q.order_by("-id") if order == "descending" else q.order_by("id")
+        if status:
+            q = q.filter(status=status)
+        return q.all()[offset:limit]
```

Comment:
Where is sorting by ID called for?  In general we've found other things like ordering by dates, names, projects etc are what users do most frequently.

Reply:
Sorting isn't called for at all right now, so I just with with the id as its a stable ordering identifier.

Reply:
If we want a stable ordering I'd just define that right on the model here instead of adding it here:

https://docs.djangoproject.com/en/3.1/ref/models/options/#ordering

Reply:
How would I reverse that if the order is changed in a query?

Reply:
Oh yeah that was my point, if we don't need to explicitly reorder ascending/descending by id, should we even bother doing it at all?  Just trying to avoid building things that aren't called for in specs/ux, but not a blocker.